BoxContentSDK/BoxContentSDK/Models/BOXItem.h

```diff
@@ -148,7 +148,7 @@
  *  Whether this item is deleted or not.
  *  Values include "active", "trashed" if the file has been moved to the trash, and "deleted" if the file has been permanently deleted
  */
-@property (nonatomic, readwrite, strong) NSString *status;
+@property (nonatomic, readwrite, copy, nullable) NSString *status;
```

Comment:
Yes, I have a ticket open to review the entire class. I will commit to unblock the upstream caller.